### PR TITLE
Start FTL in supervised mode

### PR DESCRIPTION
**Pull request by DL6ER**
## Thank you for your contribution to the Pi-hole Community! 

Please read the comments below to help us consider your Pull Request.

We are all volunteers and completing the process outlined will help us review your commits quicker.

**Please make sure you**
 1. Base your code and PRs against the repositories developmental branch.
 2. [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits as we enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions

---
- **What does this PR aim to accomplish?:**

Start FTL in supervised mode (see [FTL#1377](https://github.com/pi-hole/FTL/pull/1377))

- **How does this PR accomplish the above?:**

Add `--supervised` to the FTL call

- **What documentation changes (if any) are needed to support this PR?:**

None

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [X] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

File: advanced/Templates/pihole-FTL.service [hunk 1]
```diff
@@ -96,10 +96,11 @@ start() {
     fi
 
     if setcap CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_NET_ADMIN,CAP_SYS_NICE,CAP_IPC_LOCK,CAP_CHOWN+eip "/usr/bin/pihole-FTL"; then
-      su -s /bin/sh -c "/usr/bin/pihole-FTL" pihole
-    else
+      su "$FTLUSER" -s /bin/sh <<EOF
```
Comment (yubiuser) [inline advanced/Templates/pihole-FTL.service:99]: Do we have a `FTLUSER`?

Comment (DL6ER) [reply advanced/Templates/pihole-FTL.service]: We had one but now we don't.

**Comment by DL6ER** [comment]
Rebased on latest `development`

**Comment by DL6ER** [comment]
The failed test is expected. The test tries to start FTL from `master` branch which complains about the invalid option `--supervised`

**Review by its0x08 — APPROVED** [review]

**Comment by yubiuser** [comment]
Maybe we should be more gracefully than. Do not exit but simply ignore unknown options?

https://github.com/pi-hole/FTL/blob/5ff5bed9f86afe81ba7640170763225fa4cbbcef/src/args.c#L375

**Comment by DL6ER** [comment]
There is a POSIX recommendation that unknown options should be rejected as simply ignoring will likely cause application behavior different than you expect. `dnsmasq` behaves the same, `rm -o *` won't delete anything either (maybe you wanted `-i` = ask before every removal).

**Comment by DL6ER** [comment]
https://github.com/pi-hole/FTL/pull/1377 was closed

**Pull request closed without merging**